compliance/tests/if_type_assert/if_type_assert.go

```diff
@@ -3,9 +3,31 @@ package main
 func main() {
 	var a any
 	a = "this is a string"
+
 	if x, ok := a.(string); ok {
-		println("Expected: string")
+		println("Expected: string:", x)
 	} else {
 		println("Not Expected: should be a string")
 	}
+
```

Comment:
@paralin It seems that you missed this part of the commit, which is the relevant one for actually exposing the bug.

Reply:
@paralin here's the comment I referred to in the discussion below. you can see that this commit fixed the unused variable and added this code which is what actually causes the issue!